Sources/PointFree/Auth.swift

```diff
@@ -51,6 +51,28 @@ public func loginAndRedirect<A>(_ conn: Conn<StatusLineOpen, A>) -> IO<Conn<Resp
     |> redirect(to: .login(redirect: conn.request.url?.absoluteString))
 }
 
+public func currentUserMiddleware<A>(_ conn: Conn<StatusLineOpen, A>)
```

Comment:
this was `fetchUser` in `SiteMiddleware.swift`, I just renamed and moved it in here

Reply:
What's the deal with `currentUserMiddleware` and `fetchUser` immediately below it? Should `currentUserMiddleware` use `fetchUser`?

Reply:
yeah, `currentUserMiddleware` could just surface the `Database.User.Id`, and then compose `fetchUser` on top of it to turn it into a `Database.User?`